apps/daimo-mobile/src/view/screen/onboarding/IntroPages.tsx

```diff
@@ -26,6 +34,10 @@ export function IntroPages({
     setPageIndex(page);
   };
 
+  useEffect(() => {
+    if (useExistingPubKeyHex && useExistingStatus === "success") existingNext();
```

Comment:
we might want an `existing account found` page here where user can either join it or cancel
(if the account has money in it and only one key, then option to cancel can be disabled)

needs design though, future work

Reply:
I think this is a reasonable 80/20 for now